client/domain/CodeEvalResultObjectFactorySpec.js

```diff
@@ -22,6 +22,7 @@ describe('CodeEvalResultObjectFactory', function() {
   var CODE = "code";
   var OUTPUT = "output";
   var CORRECTNESS_TEST_RESULTS = "correctnessTestResults";
+  var LAST_TASK_RESULTS = "s";
```

Comment:
You know, something weird is going on here. I think this entire set of tests is dodgy. If you look at the existing codebase, the main "results" should not be strings, so I think some additional changes to the existing tests may be needed...